app/controllers/app.go

```diff
@@ -49,3 +76,26 @@ func (c App) Index() revel.Result {
 	}
 	return c.Render(Zone, response, rdata)
 }
+
+func init() {
```

Comment:
Not a big fan of the init() functions and I think revel has this function `https://pkg.go.dev/github.com/revel/revel#OnAppStart` which could be used instead, wdyt?

Reply:
I don't see how to use it to register the Template functions. Initial test doesn't seem to work, I probably need another pair of eyes on it.

Reply:
https://revel.github.io/manual/templates-go.html#CustomFunctions

The example shows the function being registered using a init() function.
```
func init() {
    revel.TemplateFuncs["my_eq"] = func(a, b interface{}) bool {
        return a == 0  || a == b
    }
}
```

Reply:
Works now as suggested by @lundqvistm .